src/test/java/com/epam/tc/hw5/tests/cucumber/steps/ActionStep.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.tc.hw5.tests.cucumber.steps;
+
+import com.epam.tc.hw5.tests.cucumber.context.TestContext;
+import com.epam.tc.hw5.utils.TestUsersProviderUtil;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+import io.qameta.allure.Step;
+
+public class ActionStep extends AbstractStep {
+
+    @Given("I open JDI GitHub site")
+    @Step("Open Home Page site")
+    public void openHomePage() {
+        homePage.openSite();
+    }
+
+    @Given("I login as user \"Roman Iovlev\"")
```

Comment:
You should move all of the "Given" steps into a separate class like you did for AssertionStep